homework.md

```diff
@@ -92,22 +97,22 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+It kind of saves the changes on you local version of the code.
```

Comment:
Thats correct. It worth including that it saves at a given time. Thus it is a snapshot.